services/auth/source/ldap/source_search.go

```diff
@@ -103,26 +108,47 @@ func (ls *Source) findUserDN(l *ldap.Conn, name string) (string, bool) {
 	return userDN, true
 }
 
-func dial(ls *Source) (*ldap.Conn, error) {
-	log.Trace("Dialing LDAP with security protocol (%v) without verifying: %v", ls.SecurityProtocol, ls.SkipVerify)
+func dial(source *Source) (*ldap.Conn, error) {
+	log.Trace("Dialing LDAP with security protocol (%v) without verifying: %v", source.SecurityProtocol, source.SkipVerify)
 
-	tlsCfg := &tls.Config{
-		ServerName:         ls.Host,
-		InsecureSkipVerify: ls.SkipVerify,
+	tlsConfig := &tls.Config{
+		ServerName:         source.Host,
+		InsecureSkipVerify: source.SkipVerify,
 	}
-	if ls.SecurityProtocol == SecurityProtocolLDAPS {
-		return ldap.DialTLS("tcp", fmt.Sprintf("%s:%d", ls.Host, ls.Port), tlsCfg)
+
+	if source.UseCertificate {
+		certFile := filepath.Join(setting.AppDataPath, "login-sources", source.Name+".cert.pem")
```

Comment:
It makes hard to use it, especially in dockerized environment

Reply:
but this is just a subdirectory of the data path?

Reply:
Yes, I know but you can't really manage it though docker for example, just from file system

Reply:
but this is like the rest of our certificates and keys.

Reply:
This functionality has been removed